src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+.page {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child{
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2){
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3){
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4){
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(n){
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(n) {
  background-image: url(./images/star-active.svg);
}
```

Combine selectors using comma